v4l-dvb/origin/Config.in

```diff
@@ -0,0 +1,23 @@
+choice
+    depends on PACKAGE_kmod-media
+    prompt "Used DVB (media) tree"
+    default PACKAGE_kmod-media_tbsdtv
```

Comment:
Please change this symbol, now it points to package which doesn't exist, it can be confusing. Instead use something simpler like this: `V4LDVB_TBSDTV` or `V4LDVB_SRC_TBSDTV`, but it's Your call how it'll be named. Of course change the rest accordingly.

Reply:
It has been changed accordingly to your second suggestion. I really like it.